WalletWasabi.Tests/UnitTests/Wallet/WalletBuilder.cs

```diff
@@ -57,7 +57,7 @@ public async Task<WalletWasabi.Wallets.Wallet> CreateRealWalletBasedOnTestWallet
 		keyManager.GetKeys(_ => true); // Make sure keys are asserted.
 
 		var serviceConfiguration = new ServiceConfiguration(new UriEndPoint(new Uri("http://www.nomatter.dontcare")), Money.Coins(WalletWasabi.Helpers.Constants.DefaultDustThreshold));
-		WasabiSynchronizer synchronizer = new(requestInterval: TimeSpan.FromSeconds(3), 1000, BitcoinStore, HttpClientFactory);
+		using WasabiSynchronizer synchronizer = new(period: TimeSpan.FromSeconds(3), 1000, BitcoinStore, HttpClientFactory);
```

Comment:
I don't think using the `using` keyword here is good. No problem arised from this because the tests doesn't touch the synchronizer AFAIK.

Consider making it into a property in this class and dispose it in the `DisposeAsync` func.